app/src/main/java/com/quran/labs/androidquran/ui/util/TranslationsSpinnerAdapter.java

```diff
@@ -129,4 +146,19 @@ public void updateItems(String[] translationNames,
   public interface OnSelectionChangedListener {
     void onSelectionChanged(Set<String> selectedItems);
   }
+
+  public void setOnClickListener(View.OnClickListener onClickListener) {
+    this.textListener = onClickListener;
+  }
+
+  private String[] updateTranslationNames(String[] translationNames) {
+    List<String> translationsList = new ArrayList<>();
+    for (String translation : translationNames) {
```

Comment:
ok sure thing, my only thought would be readability but ill leave the change out for now